service/resource/cloudformation/main_stack.go

```diff
@@ -0,0 +1,21 @@
+package cloudformation
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	awsCF "github.com/aws/aws-sdk-go/service/cloudformation"
+	"github.com/giantswarm/aws-operator/service/key"
+	"github.com/giantswarm/awstpr"
+)
+
+func newMainStack(customObject awstpr.CustomObject) (*awsCF.CreateStackInput, error) {
+	clusterId := key.ClusterID(customObject)
+	stackName := fmt.Sprintf("cluster-main-stack-%s", clusterId)
```

Comment:
I think we should add a helper to the key service for the stack name. Also I think the name should start with the clusterID so that stacks are ordered by cluster in the CLI / console.

Reply:
Good idea thx, done